packages/kbn-apm-synthtrace-client/src/lib/otel/index.ts

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { Fields } from '../entity';
+import { Serializable } from '../serializable';
+import { OtelError } from './error';
+import { OtelMetric } from './metric';
+import { OtelTransaction } from './transaction';
+
+interface OtelSharedResourceAttributes {
+  'service.name'?: string;
+  'agent.name'?: string;
+  'agent.version'?: string;
+  'metricset.interval'?: string;
+  'metricset.name'?: string;
```

Comment:
`metricset.name` is under `attributes`, but not  under`resource.attribute`.

So it's:
- `attributes.metricset.name`
-  `resource.attributes.metricset.interval`

(ultimately it won't make a difference, because both are queryable top level, but still this is the difference on the incoming data)

It's correct [here](https://github.com/elastic/kibana/pull/192293/files#diff-191f789d0c9edba1c311658967f64f594d8f2d35f00635742f9a3c464a738593R115).

Reply:
Moved, thanks 👍